### ISSUE TITLE
[BUG]: "The type or namespace 'Common' does not exist in the namespace 'LLama'"

### ISSUE THREAD

**Issue by crisdesivo** [SciSharp/LLamaSharp#895]
### Description

I'm trying to use LLamaSharp on unity (version 2022.3).

My OS is Ubuntu 20.04.

I've installed LLamaSharp with NuGet for Unity. Everything went fine but when trying to use it I get this error:

`error CS0234: The type or namespace name 'Common' does not exist in the namespace 'LLama' (are you missing an assembly reference?)`

Any clue what could be going on?

### Reproduction Steps

Install unity 2022.3 on ubuntu 20.04, install NuGet for Unity's latest version as of 8/9/2024, install LLamaSharp 0.15.0 trhough NuGet for Unity, try 'using LLama.Common' on a .cs file.

### Environment & Configuration

- Operating system: Ubuntu 20.04
- .NET runtime version:  Unity 2022.3 .Net
- LLamaSharp version: 0.15.0
- CUDA version (if you are using cuda backend): 12.4
- CPU & GPU device: i5-11300H, NVIDIA GeForce RTX 3050

**Comment by martindevans** [comment]
`LLama.Common` is definitely a real namespace, it's [here](https://github.com/SciSharp/LLamaSharp/tree/master/LLama/Common)!

Can you refer to any other namespaces? e.g. try `using LLama.Batched`

**Comment by crisdesivo** [comment]
LLama.Batched also prompts the same error, and so does LLama.StatefulExecutorBase. But just plain `using LLama` has no issue.

**Comment by martindevans** [comment]
Did you ever get to the bottom of this? I assume it was some kind of installation issue, but I'm not sure what.

**Comment by crisdesivo** [comment]
No, I'm still trying to solve this, I'm not sure what's going on.
